ReleaseNotes.md

```diff
@@ -1,5 +1,36 @@
 # Release Notes
 
+## New in Release 2.8.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - `nncf.quantize` signature has been changed to add `mode: Optional[nncf.QuantizationMode] = None` as its 3-rd argument, between the original `calibration_dataset` and `preset` arguments.
+  - (Common) `nncf.common.quantization.structs.QuantizationMode` has been renamed to `nncf.common.quantization.structs.QuantizationScheme`
```

Comment:
Maybe it's useful to clearly state that "It's not impacting any user API".

Reply:
In a previous NNCF version, the `nncf.QuantizationMode` with its original meaning had been marked as an API entity: 
https://github.com/openvinotoolkit/nncf/blob/aa4408b5470be7e168eb91d434be3b11e96f12a7/nncf/common/quantization/structs.py#L24-L34

in a transitive fashion since it had been used in another API entity, namely, `nncf.quantization.advanced_parameters.QuantizationParameters`:
https://github.com/openvinotoolkit/nncf/blob/aa4408b5470be7e168eb91d434be3b11e96f12a7/nncf/quantization/advanced_parameters.py#L90

In the current state the `QuantizationParameters` has incorrect/misleading documentation and type hint with respect to the same parameter due to:
https://github.com/openvinotoolkit/nncf/blob/634e6b1df142b9d6cb5d0f08acbfdffb2fa9e7c6/nncf/quantization/advanced_parameters.py#L22

and also all previous code that instantiated `QuantizationParameters` will be incorrect because `nncf.common.quantization.structs.QuantizationMode`, which you in general need for that instantiation, does not exist anymore.

No deprecations have been announced ahead of time for this change, and no time was given for the users to adjust their code. It is therefore a breaking change as stated originally.

Reply:
I guess it was missed during the review then. @AlexKoff88 , I don't think we can fix it anyhow but do you see any impact on users, optimum-intel, etc?

Reply:
No impact as these are details of internal implementation.